packages/ios_platform_images/example/integration_test/ios_platform_images_integration_test.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:ios_platform_images/ios_platform_images.dart';
+
+import 'package:ios_platform_images_example/main.dart' as app;
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets(
+    'load ios bundled image',
+    (WidgetTester tester) async {
+      app.main();
+      await tester.pumpAndSettle();
+
+      expect(find.bySemanticsLabel('Flutter logo'), findsOneWidget);
+    },
+  );
+
+  testWidgets(
+    'load ios system images',
+    (WidgetTester tester) async {
+      app.main();
+      await tester.pumpAndSettle();
+
+      expect(find.bySemanticsLabel('Smiling face'), findsOneWidget);
+      expect(find.bySemanticsLabel('Sprinting hare'), findsOneWidget);
+      expect(find.bySemanticsLabel('Ladybug'), findsOneWidget);
+    },
+  );
+
+  test(
```

Comment:
The reason CI didn't fail here turns out to be that you use `test` instead of `testWidgets`; see the Note box at https://docs.flutter.dev/testing/integration-tests#project-setup